### PR TITLE
Initial text without image

### PR DESCRIPTION
**Pull request by Ayush-Praveen**
I will add the images once design team finalizes the design changes, text can be reviewed however.

### CODE REVIEW

**Comment by Ayush-Praveen** [comment]
Added images and modified some descriptions, @Shreya-crypto , You can go ahead with the review.

File: docs/Data Lake.md [hunk 1]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+![]()
+    
+    Potential use-case example: What is the expression of SARS-CoV2 virus entry specific host protein 'ACE2' across different cell types in different studies? 
+
+![]()
+    
+    The Cell Type Aggregation tab provides the input in the form of text field where query can be made for a gene, using our internal discover services, the input gene is queried across all the datasets in which it is expressed in the repository. To check for ACE2 expression, simply enter ACE2 in the search field. After entering the gene, clicking on 'Search gene' will make a bar plot showing the median expression and the distribution of gene expression across different studies.
```
Comment (Shreya-crypto) [inline docs/Data Lake.md:394]: search gene in italics... will generate* a bar plot

**Review by Shreya-crypto — CHANGES_REQUESTED** [review]

File: docs/Data Lake.md (hunk 1 above)
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 2]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+![]()
+
+
+*   **Cell-Type Aggregation**
+
+    After having insights about different markers in a single cell study, it is imperative to have a look at the expression of the selected markers across different cell types in different studies. Single cell visualization provides pre-dataset exloratory analysis in the form of Cell Type Aggregation. Cell Type Aggregation provides the scope to query the respective repository for median expression of a selected gene across top 20 cell types based on expression. This provides the scope to study expression of a gene in a particular cell type in a given biological context. 
+
+![]()
+    
+    Potential use-case example: What is the expression of SARS-CoV2 virus entry specific host protein 'ACE2' across different cell types in different studies?
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 3]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+![]()
+    
+    The Violin plot panel is the area for exploring the marker distribution using violin plot in an interactive fashion. Using Customize Violin slider, the range of values used for plotting violin can be adjusted, it is useful for observing a section of data such as the non-zero values of the expression. 
+
+![]()
+
+
+*   **Cell-Type Aggregation**
+
+    After having insights about different markers in a single cell study, it is imperative to have a look at the expression of the selected markers across different cell types in different studies. Single cell visualization provides pre-dataset exloratory analysis in the form of Cell Type Aggregation. Cell Type Aggregation provides the scope to query the respective repository for median expression of a selected gene across top 20 cell types based on expression. This provides the scope to study expression of a gene in a particular cell type in a given biological context.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 4]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+    *Marker* tab provides exploration of distribution of markers. The tab presents, visualization methods like dot and violin plots to study the distribution of genes across different metadata. The Marker selection panel, provides the user with options to choose different genes and metadata for plotting marker distribution. 
+
+![]()
+
+    The dot plot panel, is the area for exploring the average expression and distribution of marker using dot plot in an interactive fashion.
+    
+![]()
+    
+    The Violin plot panel is the area for exploring the marker distribution using violin plot in an interactive fashion. Using Customize Violin slider, the range of values used for plotting violin can be adjusted, it is useful for observing a section of data such as the non-zero values of the expression.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 5]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+You can analyze a single dataset by selecting the checkbox to the left of the entry in the table. Once you’ve selected the checkbox, click on the *Load* button below the table description.
+
+![Select a data set](../img/Discover/image7.png)
+
+After clicking the *Load* button, the app will read the selected dataset. Once the loading finishes, you can check the further tabs to explore the dataset:
+
+*   **Dataset Summary**
+
+    The tab provides quick summary of the selected dataset. The tab presents, no. of celltype/clusters, genes and cells, available metadata and quality control metrics for the selected dataset. The Value boxes at the top provide information about the no. of genes, cells and celltypes/clusters.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 6]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+    
+*   Tissue
+
+![Search options](../img/Discover/image5.png)
+
+The search will return all datasets that are associated with your search. The result should look like the image below.
+
+![Search results](../img/Discover/image6.png)
+
+The table shown above shows very few columns by default. In order to view the other columns in the table, you can select the fields from **Available Columns** and click on *Show!* button. *Download Selected Dataset* button will let you download the dataset that you have selected on your local system. *Export results to CSV* button will let you download the search result table in the form of a .csv file. Once you have narrowed down the relevant datasets, you can analyze one or more datasets on the fly within the app.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 7]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+![Discover App](../img/Discover/single_2.png)
+
+The app will open and you should see the overview page which contains a brief **description** of the application, it's **scope** and the **usage** as shown below.
+
+![App Description](../img/Discover/image4.png)
+
+**Exploring the data lake**
+
+Search for relevant datasets by navigating to the *Dataset Search* tab in the navigation pane to the left. Keyword search can be applied to the following fields:
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 8]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+![]()       
+    
+*   **Marker**
+
+    *Marker* tab provides exploration of distribution of markers. The tab presents, visualization methods like dot and violin plots to study the distribution of genes across different metadata. The Marker selection panel, provides the user with options to choose different genes and metadata for plotting marker distribution. 
+
+![]()
+
+    The dot plot panel, is the area for exploring the average expression and distribution of marker using dot plot in an interactive fashion.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 9]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
 Single Cell Visualization provides a visualization interface for single cell data hosted on GEO. The data lake working on its backend is updated regularly with new datasets. The app has different visualizations for understanding data: *Dimensionality reduction plot*, *Violin plot*, *Dot Plot*. 
 -->
 
+##Single Cell Visualziation
+**Opening the app**
+
+Upon opening the Discover portal on Polly like the way it was opened for Discover App, choose a relevant data repository which hosts single cell data.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

File: docs/Data Lake.md [hunk 10]
```diff
@@ -276,6 +277,124 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+After selecting a repository, you’ll be able to see a dashboard and the Polly Single Cell app icon. The dashboard will give you an overview of the repository, for instance, the number of data sets in the repository, distribution of tissues, organisms, etc. among the datasets.
+
+![Repository Dashboard](../img/Discover/single_1.png)
+
+Click on the below icon to start the single cell app.
+
+![Discover App](../img/Discover/single_2.png)
+
+The app will open and you should see the overview page which contains a brief **description** of the application, it's **scope** and the **usage** as shown below.
```
[the remark this answers is not in the document]
Comment (Ayush-Praveen) [reply docs/Data Lake.md]: Done

**Comment by Ayush-Praveen** [comment]
@Shreya-crypto I have resolved your comments.

**Review by Shreya-crypto — APPROVED** [review]

**Review by Shreya-crypto — APPROVED** [review]

**Review by avijitzutshi — APPROVED** [review]

**Pull request merged**